src/test/java/com/thealgorithms/conversions/OctalToBinaryConverterTest.java

```diff
@@ -0,0 +1,15 @@
+package com.thealgorithms.conversions;
```

Comment:
Please rename the file and the class to `OctalToBinaryTest`

Reply:
Sorry for the mistake,